@ecomplus/storefront-components/src/js/APrices.js

```diff
@@ -81,7 +82,7 @@ export default {
 
     price () {
       const price = getPrice(this.product)
-      if (this.extraDiscount.value) {
+      if (this.extraDiscount.value && (price > this.extraDiscount.min_amount)) {
         return getPriceWithDiscount(price, this.extraDiscount)
       }
```

Comment:
```suggestion
      if (
        this.extraDiscount.value &&
        (!this.extraDiscount.min_amount || price > this.extraDiscount.min_amount)
      ) {
        return getPriceWithDiscount(price, this.extraDiscount)
      }
```

nem sempre o desconto tem `min_amount`

Reply:
além disso, se o intuito é previnir descontos negativos melhor editar a função `getPriceWithDiscount`, não? garantir que ela sempre retorna um valor positivo